### PR TITLE
Adjusts the files multi select header width

### LINKED ISSUE #10330: Files App: Layout issue when selecting multiple files
### Steps to reproduce
1. Open the files app
2. Go to any folder with at least one file
3. Click the checkbox before the file

### Actual behaviour
The column items in the view aren't aligned (see screenshot).
image

### Expected behaviour
The column items should be aligned.

### Server configuration
**Nextcloud version:** 14.0.0 alpha  
**Where did you install Nextcloud from:** git

### Client configuration
**Browser:** Firefox/Chrome  
**Operating system:** Ubuntu 18.04

### PR DESCRIPTION
**Pull request by weeman1337**
Just a tiny style adjustment to align the files multi select header.

closes #10330

### CODE REVIEW

File: apps/files/css/files.scss [hunk 1]
```diff
@@ -256,7 +256,7 @@ table.multiselect thead {
 	z-index: 55;
 	-moz-box-sizing: border-box;
 	box-sizing: border-box;
-	left: 250px; /* sidebar */
+	left: 300px; /* sidebar */
```
Comment (skjnldsv) [inline apps/files/css/files.scss]: Please use the proper variable :)
`$navigation-width`

**Review by skjnldsv — CHANGES_REQUESTED** [review]
Please use the proper variable :)
`$navigation-width`

**Comment by weeman1337** [comment]
@skjnldsv done. I wasn't sure if I can use it, because it's located under `core` and the other style file is under `app/files`. So the core variables can be used from any app shipped by default?

**Comment by skjnldsv** [comment]
You rocks! :)
Also, now that you're a contributor, you should be able to directly create branches from the nextcloud/server repo ;)

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
> So the core variables can be used from any app shipped by default?

Any variable used in variables.scss is appended to the compilation of the file, so you're free to use the variables directly yes :)
Note though that we deprecated the use of the scss variables for 15 in favor of css4 variables. So the variables.scss file won't be appended to the scss files anymore in the next version (15)

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Pull request merged**